### ISSUE TITLE
Use of account lib, DeprecationWarning: Buffer() is deprecated 

### ISSUE THREAD

**Issue by lathanbritz** [WietseWind/xrpl-accountlib#308]
Using this lib with evernode im getting the following warning.

im guessing this is due to the trans compilation.

`DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead`

**Comment by WietseWind** [comment]
Can you see which submodule (file path) triggers it?

**Comment by lathanbritz** [comment]
not sure how to find that but as soon as i add const lib = require('xrpl-accountlib') im getting that warning.

<img width="704" alt="Screen Shot 2022-10-17 at 17 12 27" src="https://user-images.githubusercontent.com/508629/196273880-ffec8cb9-8837-4a3d-a4c1-4bdbe9ea5c6d.png">

**Comment by WietseWind** [comment]
I checked the source of the lib. and I can't find the use of Buffer as a constructor;
```
generate/familySeed.ts:  entropy?: Buffer | Uint8Array;
utils/index.ts:function bufferToHext(buffer: Buffer): string {
utils/index.ts:  return Buffer.from(nonCanonicalDer).toString("hex").toUpperCase();
```

What kind of secret type (mnemonic, family seed, ... ) are you using when you get this error?

**Comment by lathanbritz** [comment]
<img width="494" alt="Screen Shot 2022-10-17 at 21 47 46" src="https://user-images.githubusercontent.com/508629/196309651-ff56b9e8-0bf6-4391-9732-2165f8d9910e.png">
thats the thing, i dont even need to have` accountSetup(ctx) ` and it throws the warning as soon as i add `const lib = require('xrpl-accountlib')` don't even use it just declare it the warning is printed from the contract.

**Comment by lathanbritz** [comment]
figured out this is part of Evernodes issue after being able to dump the stack trace.

**Comment by lathanbritz** [comment]
FYI, Found the source of this one @WietseWind its the secret libs codec thats used.

**Comment by WietseWind** [comment]
Created a PR: https://github.com/professorhantzen/ripple-secret-codec/pull/2

If merged & published to npm: great, otherwise I'll incorporate the updated version in `xrpl-accountlib` soon.

**Comment by WietseWind** [comment]
Merged :) Updated ripple-secret-codec@1.0.3 is now included in the just published `xrpl-accountlib@1.5.3` :)
